packages/xcm-util/convert-xcm-version/upgrade-xcm-version.ts

```diff
@@ -221,6 +299,31 @@ function upgradeNetworkIdV2(networkId: NetworkIdV2): NetworkIdV3 | null {
   }
 }
 
+function upgradeNetworkIdV4(networkId: NetworkIdV3): NetworkIdV5 {
+  if (networkId === 'westend') {
+    return {
+      byGenesis:
+        '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e',
+    };
+  }
```

Comment:
We must check that this is really working. I don't know if PolkadotJS can encode these hexes directly.
We may need to sanitize them before giving the XCM object to the PolkadotJS.

For instance, PJS can't convert text-represented account addresses inside the XCM objects.